.babelrc

```diff
@@ -21,27 +20,20 @@
         }
       }
     ],
-    "babel-preset-react" // transform JSX to JS
+    "@babel/preset-react" // transform JSX to JS
   ],
   "plugins": [
+    "@babel/plugin-transform-modules-commonjs", // allows module.exports at the helpers/logger.*.js
+    "@babel/plugin-proposal-object-rest-spread", // allows ...spread notation
+    "@babel/plugin-syntax-dynamic-import", // allows `await import()`
+    "dynamic-import-node", // allows `return import()`
+    "@babel/plugin-transform-runtime", // prevent duplication of Babel helper functions in output
     [
       "babel-plugin-styled-components",
       {
         "ssr": true, // avoid checksum mismatches (different class generation between client & server)
         "fileName": false // prevent filename forming part of class name (duplication)
       }
-    ],
-    "babel-plugin-syntax-dynamic-import", // allows `await import()`
-    "babel-plugin-dynamic-import-node", // allows `return import()`
-    "transform-object-rest-spread", // allows ...spread notation
-    [
-      // polyfills code without polluting globals
-      // https://stackoverflow.com/a/36821986
-      "transform-runtime",
-      {
-        "polyfill": false,
```

Comment:
This is no longer an option in @babel/plugin-transform-runtime (and now defaults to `true` under the hood).